packages/datagateway-download/src/downloadStatus/adminDownloadStatusTable.component.tsx

```diff
@@ -39,6 +39,8 @@ import {
 } from '@material-ui/icons';
 import RefreshIcon from '@material-ui/icons/Refresh';
 import BlackTooltip from '../tooltip.component';
+import { toDate } from 'date-fns-tz';
+import { format } from 'date-fns';
```

Comment:
What was the reason for not using the `format` function from `date-fns-tz`? I couldn't notice any difference when I tried it out. I'm going to add a commit which removes the `date-fns` library and approve the PR because I'm happy with it anyway but if I'm missing something underlying with the timezones, feel free to revert that commit.

Reply:
You can use the format function from `date-fns-tz`, but we don't actually need the functionality of that library for the `format` function (aka appending timezone information strings). If we wanted to display timezone info in the table I think scientists would probably just prefer UTC offsets 😅  we do need to add `date-fns` to `package.json` as `date-fns-tz` depends on it so we don't actually reduce any overhead since we are already implicitly importing `format` from `date-fns` (as that's what `date-fns-tz` uses) - in fact using the `date-fns-tz` one would actually increase our footprint I think since the function does "more stuff" and so is larger

Reply:
That makes a lot of sense! In which case feel free to revert my commit (or let me know if you want me to do it)